src/actions/recomputeReduxState.js

```diff
@@ -66,6 +66,9 @@ const modifyStateViaURLQuery = (state, query) => {
   if (query.c) {
     state["colorBy"] = query.c;
   }
+  if (query.focus === "selected") {
+    state["focus"] = true;
+  }
```

Comment:
Within the state recomputation we need to check if `query.focus` is set to something other than "selected" and remove it if so. With the code as written, loading `?focus=foo` won't set the focus (good!) but will leave it in the URL query, and then if we toggle on focus the URL middleware (as currently written) will remove the URL query rather than setting it to `?focus=selected`.